javalib/src/main/scala/java/util/PriorityQueue.scala

```diff
@@ -0,0 +1,125 @@
+package java.util
+
+import java.lang.Comparable
+import scala.math.Ordering
+import scala.collection.mutable
+import scala.annotation.tailrec
+import scala.language.existentials
+
+class PriorityQueue[E] protected (ordering: Ordering[_ >: E], _comparator: Comparator[_ >: E])
+    extends AbstractQueue[E]
+    with Serializable { self =>
+
+  def this(initialCapacity: Int) = {
+    this(defaultOrdering[E], null.asInstanceOf[Comparator[_ >: E]])
+    if (initialCapacity < 1) throw new IllegalArgumentException()
+  }
+
+  def this() =
+    this(11)
+
+  def this(initialCapacity: Int, comparator: Comparator[_ >: E]) = {
+    this(PriorityQueue.safeGetOrdering[E](comparator), null.asInstanceOf[Comparator[E]])
+    if (initialCapacity < 1) throw new IllegalArgumentException()
+  }
+
+  def this(c: Collection[_ <: E]) = {
+    this(defaultOrdering[E], null.asInstanceOf[Comparator[E]])
+    addAll(c)
+  }
+
+  def this(c: PriorityQueue[_ <: E]) = {
+    this(PriorityQueue.safeGetOrdering[E](c.comparator()), c.comparator().asInstanceOf[Comparator[E]])
```

Comment:
This line is too long for no good reason. You can break it before the second argument.